src/files-and-videos/videos-page/transcript-settings/TranscriptSettings.test.jsx

```diff
@@ -3,6 +3,7 @@ import {
   act,
```

Comment:
Looking at this file, I can't see a test case which matches the issue described in the ticket. Could you add a 

"Now the correct option is selected when the settings are loaded" Test? That would improve my confidence in what this PR is trying to do.